### DISCUSSION TITLE
Uploading Files Directly Into Spigot.com

### DISCUSSION THREAD

**Discussion by asfouralbeirut** [TownyAdvanced/Towny#6019] [General]
Hi, I tried to suggest your plugin to aternos.com and I received this message
![mceclip4](https://user-images.githubusercontent.com/104522300/178815183-a5e971cc-12b2-4b10-b514-190887fd2040.png)
so I need you to upload your files directly, not via external site. Then I'm sure you will reach more people and I wish for that. Thanks in advance.

**Comment by Pyeongher** [comment]
agree

**Comment by barissln0** [comment]
Logical! I agree ^^

**Comment by LlmDl** [comment]
When I get to 400 sponsors I will upload it to Spigot.

**Comment by xiduckz** [comment]
sehr richtig

**Comment by iEnesReisQ** [comment]
Nicee >^<

**Comment by manul694** [comment]
just buy cheap hosting and stop suffering from aternos

**Comment by ggvbo** [comment]
This was addresed in https://github.com/TownyAdvanced/Towny/discussions/5240#discussioncomment-1248044
